app/processing/statement/get-statement.js

```diff
@@ -2,15 +2,15 @@ const db = require('../../data')
 const { getDetails, getAddress, getDetailedFunding, getScheme, getDetailedPayments } = require('./components')
 const getCalculation = require('../calculation')
 const getPaymentRequest = require('../payment-request')
-const getSettlement = require('../settlement')
+const { getSettlement } = require('../settlement')
 
 const getStatement = async (settlementId) => {
   const transaction = await db.sequelize.transaction()
   try {
     const settlement = await getSettlement(settlementId, transaction)
     const paymentRequestId = settlement.paymentRequestId
     const paymentRequest = await getPaymentRequest(paymentRequestId, transaction)
-    const calculation = await getCalculation(paymentRequestId, transaction)
+    const calculation = await getCalculation(paymentRequest, transaction)

```

Comment:
Same comment as before, can we be explicit w/ what `getCalculation` and `getDetailedPayments` take as params

Reply:
Moved to https://eaflood.atlassian.net/browse/PAY-2365